packages/core/content-manager/admin/src/pages/EditView/components/DocumentActions.tsx

```diff
@@ -742,14 +745,35 @@ const PublishAction: DocumentActionComponent = ({
         status: 'published',
       });
       if (errors) {
-        toggleNotification({
-          type: 'danger',
-          message: formatMessage({
-            id: 'content-manager.validation.error',
-            defaultMessage:
-              'There are validation errors in your document. Please fix them before saving.',
-          }),
-        });
+        let hasUnreadableRequiredField = false;
```

Comment:
Without a more significant refactor of this component, **I think the only scenario we can cover with a new specific error message here is when the user cannot read a required field.**

All the publishAction logic is based off "formValues". That is whats seen in the frontend form and what is sent to the publish action in the end.

Even if the draft in the DB has a value in the required field, if the user cannot read that field then it's value in form values will be null/undefined.

So we should always error with a message that says something like "There is a required field without a value". Even if in the DB this field does hold a value (this component doesn't know about that DB value). From the admin users perspective it doesn't have a value (they can't read the field).